octopusdeploy/resource_machine.go

```diff
@@ -41,6 +42,7 @@ func resourceMachine() *schema.Resource {
 								"AzureWebApp",
 								"Ftp",
 								"AzureCloudService",
+								"Kubernetes"
```

Comment:
Before 1.0 I think we should make these first class. i.e. have a `machine_kuberneties` element instead